lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -36,6 +40,21 @@ export async function checkOnboardingBranch(
   setGitAuthor(config.gitAuthor);
   const onboardingPr = await getOnboardingPr(config);
   if (onboardingPr) {
+    isConflicted = await scm.isBranchConflicted(
+      config.baseBranch!,
+      config.onboardingBranch!
+    );
+    if (isConflicted) {
+      // if branch is conflcted ensure comment
+      await ensureComment({
+        number: onboardingPr.number,
+        topic: 'Branch Conflicted',
+        content: emojify(
+          `:warning: This PR has a merge conflict. However, Renovate is unable to automatically fix that due to edits in this branch. Please resolve the merge conflict manually.\n\n`
+        ),
+      });
+    }
+
     if (config.onboardingRebaseCheckbox) {
       handleOnboardingManualRebase(onboardingPr);
```

Comment:
Should this clear any conflict comment which exists?

Reply:
If box is checked user wants a pr-update so we should update the pr-body but this shouldn't care about the conflict-state of the branch.